### PR TITLE
Include plugin version in info message

### PR DESCRIPTION
**Pull request by Hanzei**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -53,7 +54,6 @@ bundle:
 	rm -rf dist/
 	mkdir -p dist/$(PLUGIN_ID)
 	cp $(MANIFEST_FILE) dist/$(PLUGIN_ID)/
-	cp assets/logo_dark.png dist/$(PLUGIN_ID)/
```
Comment (kaakaa) [inline Makefile]: Why did this line delete?

File: Makefile [hunk 2]
```diff
@@ -4,6 +4,7 @@ NPM ?= $(shell command -v npm 2> /dev/null)
 HTTP ?= $(shell command -v http 2> /dev/null)
 CURL ?= $(shell command -v curl 2> /dev/null)
 MANIFEST_FILE ?= plugin.json
+BUNDLE_NAME = $(PLUGIN_ID)_$(PLUGIN_VERSION).tar.gz
 
 # Verify environment, and define PLUGIN_ID, HAS_SERVER and HAS_WEBAPP as needed.
```
Comment (kaakaa) [inline Makefile]: Should add `PLUGIN_VERSION` in this comments.

**Comment by Hanzei** [comment]
Thanks for the review! All your comments are valid. Can you give this a second round of review?

File: Makefile [hunk 3]
```diff
@@ -82,21 +83,21 @@ ifneq ($(and $(MM_SERVICESETTINGS_SITEURL),$(MM_ADMIN_USERNAME),$(MM_ADMIN_PASSW
...
 	@curl -s -H "Authorization: Bearer $(TOKEN)" -X DELETE $(MM_SERVICESETTINGS_SITEURL)/api/v4/plugins/$(PLUGIN_ID) > /dev/null
-	@curl -s -H "Authorization: Bearer $(TOKEN)" -X POST $(MM_SERVICESETTINGS_SITEURL)/api/v4/plugins -F "plugin=@dist/$(PLUGIN_ID).tar.gz" > /dev/null && \
+	@curl -s -H "Authorization: Bearer $(TOKEN)" -X POST $(MM_SERVICESETTINGS_SITEURL)/api/v4/plugins -F "plugin=@dist/$(BUNDLE_NAME)" > /dev/null && \
 		curl -s -H "Authorization: Bearer $(TOKEN)" -X POST $(MM_SERVICESETTINGS_SITEURL)/api/v4/plugins/$(PLUGIN_ID)/enable > /dev/null && \
 		echo "OK." || echo "Sorry, something went wrong."
 else ifneq ($(wildcard ../mattermost-server/.*),)
 	@echo "Installing plugin via filesystem. Server restart and manual plugin enabling required"
 	mkdir -p ../mattermost-server/plugins
-	tar -C ../mattermost-server/plugins -zxvf dist/$(PLUGIN_ID).tar.gz
+	tar -C ../mattermost-server/plugins -zxvf dist/$(BUNDLE_NAME).tar.gz
```
Comment (kaakaa) [inline Makefile]: whoops! BUNDLE_NAME includes .tar.gz, so now it is duplicated.

Comment (Hanzei) [reply Makefile]: Fixes. Today is not my most concentrated day...

Comment (kaakaa) [reply Makefile]: We have worked hard for now.
It sometimes happens :-)

**Pull request merged**